bundler/man/bundle-config.1.txt

```diff
@@ -70,43 +70,44 @@ REMEMBERING OPTIONS
 
        However,  this will be changed in bundler 3, so it's better not to rely
        on this behavior. If these options must be remembered, it's  better  to
-       set them using bundle config (e.g., bundle config set path foo).
+       set  them  using  bundle  config  (e.g., bundle config set --local path
+       foo).
 
        The options that can be configured are:
 
-       bin    Creates   a   directory   (defaults  to  ~/bin)  and  place  any
+       bin    Creates  a  directory  (defaults  to  ~/bin)   and   place   any
               executables  from  the  gem  there.  These  executables  run  in
               Bundler's context. If used, you might add this directory to your
-              environment's PATH variable. For  instance,  if  the  rails  gem
+              environment's  PATH  variable.  For  instance,  if the rails gem
               comes with a rails executable, this flag will create a bin/rails
-              executable that ensures that all referred dependencies  will  be
+              executable  that  ensures that all referred dependencies will be
```

Comment:
My `groff` seems to generate unnecessary changes like here, could someone run `rake man:build` with a version that doesn't cause so many changes? Then I'll pick it up in this PR.

Reply:
Yes, I hate it :angry:. My `groff` version does that too. I think we might want to pass a ruby script over the result of `groff` (instead of just `col -xb`) to eliminate these whitespace differences.

Reply:
I made https://github.com/eregon/rubygems/commit/ad68c24e09e812b906dbe4a1542ac33f6d7482e8 that fixes the extra `1m` and `22m` in the diff:
```diff
-NAME
-       bundle-add - Add gem to the Gemfile and run bundle install
+1mNAME0m
+       1mbundle-add 22m- Add gem to the Gemfile and run bundle install
-SYNOPSIS
-       bundle     add     GEM_NAME     [--group=GROUP]     [--version=VERSION]
+1mSYNOPSIS0m
+       1mbundle     add     4m22mGEM_NAME24m     [--group=GROUP]     [--version=VERSION]
        [--source=SOURCE]   [--git=GIT]   [--branch=BRANCH]    [--skip-install]
```
but it does not fix those whitespace changes. So I wonder if `groff` somehow forgot to strip those escape sequences and that causes the extra spacing changes.

Reply:
BTW, what are the `.txt` for? Isn't the `.ronn` enough for "readable without `man`"?

Reply:
> but it does not fix those whitespace changes. So I wonder if groff somehow forgot to strip those escape sequences and that causes the extra spacing changes.

I think the extra spacing is generated due to `ronn` generating TABs at random places when you make changes to a ronn source. But not really sure.

> BTW, what are the .txt for? Isn't the .ronn enough for "readable without man"?

That's a great point. We can probably print the `ronn` sources when `man` is not available and stop generating these txt files.

Reply:
I ended up using `git add -p`, and looking at which lines were relevant with `git diff -w`. Quite bothersome but it worked.